healthcheck.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# Check if `socat` is running
+if [[ -z $(pidof socat) ]]; then
```

Comment:
shellcheck complains about this line when I was testing locally. This seems to work:

```suggestion
if [ -z "$(pgrep socat)" ]; then
```

(You can use pidof as well, I don't have it on OSX)